docs/data/material/pages.ts

```diff
@@ -297,6 +310,18 @@ const pages: MuiPage[] = [
       },
     ],
   },
+  {
+    pathname: '/material-ui/discover-more',
+    children: [
+      { pathname: '/material-ui/discover-more/showcase' },
```

Comment:
**Off-topic**

The showcase is so buried in the content that it feels it defeats the purpose of the page. I think https://nextjs.org/ has it right, it's one of the first things we want new users to check to build up trust with them.

<img src="https://github.com/mui/material-ui/assets/3165635/e8d05899-a62d-436f-98ca-cd979691630d" width="682">

Reply:
Agreed. Their structure is a bit different, though (i.e., marketing site links persist in the navbar when transitioning to the documentation). It's a perk of having a domain per product, in my perspective. In any case, maybe the Showcases page could be moved over to the Getting started. There might be an argument to be made that quickly seeing projects developed with the library helps people make a decision and commit to it.

Reply:
I was thinking of having the showcase link visible from https://mui.com/material-ui/ page, maybe like done here:

![Screenshot 2024-01-16 at 21 35 05](https://github.com/mui/material-ui/assets/3165635/99197344-7e20-44dd-a050-115b11b58cb7)